tests/integration-tests/helix-payload/osx-helix-payload.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+version='1.0.0-ci'
+
+# Clean the NuGet cache from the previous 1.0.0-ci version of the tool
+# TODO: This might have a better solution: https://github.com/dotnet/xharness/issues/123
+echo "Cleaning the NuGet cache from the previous version of the tool..."
+cache_dirs=`dotnet nuget locals All -l | cut -d':' -f 2 | tr -d ' '`
+while IFS= read -r path; do
+    rm -rf "$path/microsoft.dotnet.xharness.cli/$version"
+    rm -rf "$path/Microsoft.DotNet.XHarness.CLI/$version"
+done <<< "$cache_dirs"
+
+set -x
+
+here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# TODO - Package the app bundle here using `dotnet xharness ios package ..`
+# For now we download a pre-bundled app:
+curl -L --output $here/app.zip "https://netcorenativeassets.blob.core.windows.net/resource-packages/external/macos/test-ios-app/System.Numerics.Vectors.Tests.app.zip?sp=r&st=2020-05-04T13:23:20Z&se=2028-05-04T21:23:20Z&spr=https&sv=2019-10-10&sr=b&sig=7sKBDMZrlk%2FA58zlbaUYptb98kK7EacQpmJ9RxlLLrE%3D"
```

Comment:
SAS token is unnecessary on this URL (resource-packages is a publicly readable container)